complete/src/main/java/com/example/restservice/service/QueueService.java

```diff
@@ -24,8 +25,14 @@
   @Autowired private UserService userService; // TODO remove
 
   public CreateQueueResponse createQueue(CreateQueueRequest createQueueRequest) {
-    var queue = queueRepository.save(new Queue(createQueueRequest.getQueueName()));
-    return new CreateQueueResponse(queue.getQueueName(), queue.getQueueId());
+    try {
```

Comment:
I'm not familiar with the latest java features. Any cleaner way of doing this?

Reply:
This looks good to me, just that we don't need the second catch-all. We can let that exception flow down, and then spring would catch it an respond with an internal server error.